docs/antora-site-structure.md

```diff
@@ -31,41 +31,56 @@ The main reasons for using [Antora][link-antora] is the following:
+
 
 ```
-layer-1     layer-2
-master      included repos
+main        content source
 
             docs-client-desktop
 docs    --> docs-client-ios-app 
             docs-client-android
             ...
 
 ```
-Note that the arrow from level-1 to level-2 is intentionally unidirectional and should be respected. See more details about the reason below. 
+Note that the arrow from main to content source is intentionally unidirectional in our setup and should be respected. See more details about the reason below. 
+
+### Scope of the playbook files (site.yml)
+
+In general, only one `site.yml` file is neccessary for the whole environment and its definitions are **available to the whole site**. This `site.yml` file is located in the main repo `docs`. We have additionally for each content source its own `site.yml` for testing purposes only. The scope of these local `site.yml` files is restricted to the respective content source and any definitions made are not availabe outside. 
+
+Due to this fact, you need to re-add all relevant attributes of the main `site.yml` file in the `site.yml` file of the content source which accesses it, else a local build will return warnings about unresolved attributes.
+
+If you have added an attribute in a content source `site.yml` file, you must add this attibute to the main `site.yml` file to avoid a build warning (unresolved attribute) during a build of the entire documentation.
+
+Note that this behaviour is relevant for the playbook `site.yml` files only and does not apply to the component descriptor files `antora.yml`.
 
 ## Scope of Content Accessibility
 
-Because Antora is capable of defining additional resources, you can access content from these resources. To do so, follow the [Resource ID Coordinates][link-resource-id] scheme.
+Because Antora is capable of defining additional content sources, you can access content from these resources. The setup is flat, no main/child environment. To access ressources, follow the [Resource ID Coordinates][link-resource-id] scheme.
+
+Because the setup we have made regarding testing, the direction of the arrow is important.
+
 
 **Possible**
 ```
 docs    --> docs-client-ios-app
             └> index.adoc
 ```
-Note that the direction of the arrow is important.
-
-Level-1 (docs) can access content from level-2 (docs-client-ios-app) _at any time_ because docs has referenced the repo (content source).
-
-Level-2 (docs-client-ios-app) can reference content from level-1 (docs) only during a level-1 build and not during a level-2 build (eg a local test build). This is to avoid circular content sourcing. Depending on the type of content being accessed from level-1, a warning or an error will be thrown. To overcome this situation, use .html references to level-1.
+Main (docs) can access content from any conten source (like docs-client-ios-app) and vice versa *at build time* , because docs has referenced the repo (content source) and made it available to all.
```

Comment:
```suggestion
Main (docs) can access content from any content source (like docs-client-ios-app) and vice versa *at build time* , because docs has referenced the repo (content source) and made it available to all.
```